src/tools/run-make-support/src/diff/mod.rs

```diff
@@ -87,9 +87,7 @@ impl Diff {
         self
     }
 
-    #[track_caller]
-    pub fn run(&mut self) {
-        self.drop_bomb.defuse();
+    fn run_common(&self) -> (&str, &str, String, String) {
```

Comment:
Would be better to return a struct with named field instead of a tuple here.

Reply:
Hum although it's a private function, so not really a big issue either...

Reply:
Yeah I was also about to comment on that but I also realized that's private lol